tests/10-vcdm2.js

```diff
@@ -97,20 +98,25 @@ describe('Verifiable Credentials Data Model v2.0', function() {
       return result;
     }
 
-    async function verifyVp(vp) {
+    async function verifyVp(vp, options = {checks: []}) {
       const body = {
         verifiablePresentation: vp,
-        options: {
-          checks: ['proof'],
-        }
+        options
       };
       const result = await post(vpVerifier, body);
-      if(result.errors.length) {
+      if(result?.errors?.length) {
         throw result.errors[0];
       }
       return result;
     }
 
+    const challenge = 'z19pnWMzs2Ub452WW9BLBJP9h';
+    const proveOptions = {challenge};
+    const verifyPresentationOptions = {
+      checks: ['proof'],
```

Comment:
It feels wrong to not at least check the proof when verifying. What happens if `proof` isn't listed in the checks, or the checks are empty? If the answer is: success, then we need to discuss.

Reply:
This was discussed internally and was considered to be ok.

Reply:
What is happening internally is of no consequence to the other implementers. This is something that other implementers might feel that they have to implement, so we have to think more holistically about if this is required for other implementers. At present, I guess the answer is "no, it's not required"... we can leave it for now, but should document that we're looking for implementer feedback on the option.

Reply:
We are checking the proof when verifying if that helps to assuage any concerns and proof if the check we are requiring currently for VPs.